docs/vc-rest/w3c_api_impl_status.md

```diff
@@ -0,0 +1,43 @@
+# W3C CCG HTTP API Implementation Status
+
+The TrustBloc VC edge service supports multiple W3C CCG HTTP APIs. This page provides the status of the implementation. For detailed API documentation, refer [TrustBloc VC Service Open API Spec](openapi_demo.md).
```

Comment:
(and also Transmute API)

Reply:
[Done](https://github.com/trustbloc/edge-service/compare/fcd5f59e27bafee8e169ab9bdf2ee79bc8731c19..c78c913a8ef52721263c5eb7ce3b4d2ec4024a41#diff-e3e8ccc1e3586fbcfa5cd223b35fcbdbR3)